template.rb

```diff
@@ -246,6 +247,15 @@ def render_file(filename)
     require "open-uri"
 
   CODE
+  
+  initializer 'delegation_monkey_patch.rb', <<-RUBY
+    module ActiveRecord
```

Comment:
I'm not familiar with how templates work. Is this code (form the `delegation_monkey_patch.rb` file) basically automatically compiled into this file? Otherwise, I don't understand why the exact same code is in two places.

Reply:
@LoganDSPrice Ah, there is some slight duplication. `initializer` creates an file in the `config/initializers` folder with the content of the text passed to it. I _should_ be able to do something similar to file `"config/initializers/delegation_monkey_patch.rb", render_file("delegation_monkey_patch.rb")` but the `render_file` method was having issues so I opted to use this other method.

Reply:
I think a different issue I need to fix is that `render_file` is failing to look into files that are saved locally and so it tries to find the file on the master branch of the repo, but since this isn't the master branch, it can't find the file. Once a feature is added this isn't an issue but since this is still a branch 
```
file "config/initializers/delegation_monkey_patch.rb", render_file("delegation_monkey_patch.rb")
```
will fail.

Reply:
@raghubetina I originally tried to use the file in the `/files` folder and render it but it fails on non-master branches. Do you have preference over which method is used?

Reply:
It fails on non-master branches? That's... odd.

In the long-run it seems cleaner to have files being copied rather than a bunch of heredocs in `template.rb`, so it feels like maybe we should move in that direction — what do you think?

Reply:
@raghubetina Since the `render_file` method uses a method, `path_to_blob` that uses this as a path 
```
"https://raw.githubusercontent.com/firstdraft/appdev_template/master/files/#{filename}"
```
open-uri will throw an error if the file doesn' t exist on the master branch.

![image](https://user-images.githubusercontent.com/17581658/58128410-2e857d80-7bdd-11e9-9022-192eb5c1dc6c.png)

Yes, but I agree rendering files is better than using a lot of heredocs. I think I'll pause this pr until I fix the issue of rendering files not on the master branch.